src/Autofac.Extras.Moq/AutoMock.cs

```diff
@@ -1,5 +1,5 @@
 // This software is part of the Autofac IoC container
-// Copyright (c) 2013 Autofac Contributors
+// Copyright (c) 2020 Autofac Contributors
```

Comment:
This should either stay 2013 (first year of publication) or be a range (2013-2020). [See explanation of copyright date here.](https://smallbusiness.chron.com/year-copyright-statement-mean-62027.html)

Reply:
Reverted